### DISCUSSION TITLE
1.0.2

### DISCUSSION THREAD

**Discussion by firstof9** [firstof9/lovelace-grocy-meal-plan-card#15] [General]
# :sparkles: New Features :sparkles:
Adds 'section' to the displayed meal.

![image](https://user-images.githubusercontent.com/1105672/184174161-a873b8ef-04c9-462a-b1dc-cf1d819ef2b7.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/firstof9/lovelace-grocy-meal-plan-card/releases/tag/1.0.2'>1.0.2</a>.</em>

**Comment by Deejayfool** [comment]
Thank you for this really fast update ! ❤️

**Reply by firstof9** [reply]
No problem.
